meta-lmp-base/recipes-kernel/wireguard/wireguard-tools_%.bbappend

```diff
@@ -0,0 +1,20 @@
+# Upstream-status: backport only for kirkstone [remove for scarthgap]
+# not supposed to be upstream on kirkstone
+# pending for master/scarthgap
+# https://lists.openembedded.org/g/openembedded-devel/message/105808
+
+do_install () {
+    oe_runmake DESTDIR="${D}" PREFIX="${prefix}" SYSCONFDIR="${sysconfdir}" \
+        SYSTEMDUNITDIR="${systemd_system_unitdir}" \
+        WITH_SYSTEMDUNITS=${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'yes', '', d)} \
+        ${PACKAGECONFIG_ARGS} \
+        install
+}
+PACKAGECONFIG ??= "bashcompletion wgquick"
+
+PACKAGECONFIG[bashcompletion] = "WITH_BASHCOMPLETION=yes, , , , , "
+PACKAGECONFIG[wgquick] = "WITH_WGQUICK=yes, , ,${PN} bash,${PN}-wg-quick, "
```

Comment:
Please see lines 19-31 of this Makefile:
`https://github.com/WireGuard/wireguard-tools/blob/master/src/Makefile`

We cannot just leave variables empty. We must set `WITH_BASHCOMPLETION=no` and `WITH_WGQUICK=no` to explicitly disable these features.

Reply:
ok, I have tested and the result satisfied me, but it's easy to add the =no so let me change this now